arcgis-ios-sdk-samples/Maps/Download preplanned map/PreplannedMapAreaTableViewCell.swift

```diff
@@ -19,7 +19,7 @@ import UIKit
 // MARK: - Constants
 
 private enum Constants {
-    static let progressTintColor = UIColor.primaryBlue
+    static let progressTintColor = UIColor.primaryThemeColor
```

Comment:
This shouldn't be needed anymore since the `tintColor` of `UIProgressView`'s appearance proxy is now set in the app delegate.